ApolloExtra.lua

```diff
@@ -2469,67 +2591,153 @@ OlympusLootData.ApolloUpgrade = {
 				RequiredGodLoot = "ArtemisUpgrade",
-				{ Cue = "/VO/Apollo_0047",
+				{ Cue = "/VO/Apollo_0066",
 					StartSound = "/Leftovers/World Sounds/MapZoomInShort", UseEventEndSound = true,
-					Text = "The sullen Artemis reached out to you, did she? Oh, that's adorable, dearest! Just don't make any sudden movements with her, {#DialogueItalicFormat}hm{#PreviousFormat}?" },
+					Text = "My sister is a bit antisocial at times... I tried to get her off her shell... Even the Great Apollo can't do anything. Perhaps you can help her with that, Zagzag?" },
 			},
-			ApolloAboutDionysus01 =
+			ApolloAboutArtemis02 =
 			{
-                Name = "ApolloAboutDionysus01",
+                Name = "ApolloAboutArtemis02",
 				PlayOnce = true,
 				RequiredFalseTextLinesThisRun = GameData.GodAboutGodVoiceLines,
 				PreEventFunctionName = "BoonInteractPresentation", PreEventFunctionArgs = { PickupWait = 1.0, },
-				RequiredTextLines = { "ZeusFirstPickUp", "PoseidonFirstPickUp", "AthenaFirstPickUp", "AphroditeFirstPickUp", "AresFirstPickUp", "ArtemisFirstPickUp", "DionysusFirstPickUp", "ApolloFirstPickUp" },
-				RequiredGodLoot = "DionysusUpgrade",
-				{ Cue = "/VO/Apollo_0048",
+				RequiredTextLines = { "ApolloAboutArtemis01" },
+				RequiredGodLoot = "ArtemisUpgrade",
+				{ Cue = "/VO/Apollo_0067",
 					StartSound = "/Leftovers/World Sounds/MapZoomInShort", UseEventEndSound = true,
-					Text = "You are so fortunate to have received the Boon of Dionysus prior to my own! He, the most noble and important of us all, the god of wine, himself!" },
+					Text = "Artemis is older than me by seconds, but she loves to make me feel like the youngest. I mean by seconds... That doesn't count right?" },
 			},
-			ApolloAboutDemeter01 =
+			ApolloAboutArtemis03 =
 			{
-                Name = "ApolloAboutDemeter01",
+                Name = "ApolloAboutArtemis03",
 				PlayOnce = true,
 				RequiredFalseTextLinesThisRun = GameData.GodAboutGodVoiceLines,
 				PreEventFunctionName = "BoonInteractPresentation", PreEventFunctionArgs = { PickupWait = 1.0, },
-				RequiredTextLines = { "ZeusFirstPickUp", "PoseidonFirstPickUp", "AthenaFirstPickUp", "AphroditeFirstPickUp", "AresFirstPickUp", "ArtemisFirstPickUp", "DionysusFirstPickUp", "ApolloFirstPickUp" },
-				RequiredFalseTextLines = { "DemeterLiftsWinterCurse01" },
-				RequiredGodLoot = "DemeterUpgrade",
-				{ Cue = "/VO/Apollo_0181",
+				RequiredTextLines = { "ApolloAboutArtemis01" },
+				RequiredGodLoot = "ArtemisUpgrade",
+				{ Cue = "/VO/Apollo_0068",
 					StartSound = "/Leftovers/World Sounds/MapZoomInShort", UseEventEndSound = true,
-					Text = "Why, did sorrowful old Demeter get all the way to you before I did? She stubbornly refuses me when I offer to help! Unlike you, little godling!" },
+					Text = "I wonder if I could beat Artemis in a bow competition... We are both pretty good... Tell me, Zagzag... You would bet on me, right?" },
```

Comment:
"I wonder if I could beat Artemis in a bow competition. We are both pretty good, so... Tell me, Zagzag... You would bet on me, right?"